floris/floris_model.py

```diff
@@ -942,54 +943,65 @@ def get_turbine_TIs(self) -> NDArrayFloat:
 
     ### Methods for sampling and visualization
 
+    def set_for_viz(self, findex: int, solver_settings: dict) -> None:
+        """
+        Set the floris object to a single findex for visualization.
+
+        Args:
+            findex (int): The findex to set the floris object to.
+            solver_settings (dict): The solver settings to use for visualization.
+        """
+        # TODO: can get from properties once #843 is merged
```

Comment:
Is this resolved or still relevant?

Reply:
Good catch, thanks---I've addressed this now, as well as a couple of other TODOs.